docs/oid-info.md

```diff
@@ -2,52 +2,91 @@
 
 ## Description
 
-Sigstore maintains its own Private Enterprise Number (57264) with the Internet
+Sigstore maintains its own Private Enterprise Number ([57264](http://oid-info.com/get/1.3.6.1.4.1.57264)) with the Internet
 Assigned Numbers Authority to help identify and organize additional metadata in
 code signing certificates issued by Fulcio instances. This document aims to
 provide a simple directory of values in use with an explanation of their
 meaning.
 
+## Requirements to support signing with CI/CD workload identities
+
+In order to support Sigstore code signing with CI/CD based workfload identities the following claims must be included in the OIDC ID Token. See example claim values for each extension in the detailed [Directory](#directory). 
+
+Providers MAY choose to emit extension value in other formats to generic examples, and consumers MUST NOT assume the generic example format.
+
+Requirements:
+
+- MUST include `iss` claim for `Issuer` extension.
+- MUST include claim to support: `Build Signer URI` that identifies the specific build instructions that are responsible for signing.
+- MUST include claim to support: `Build Signer Digest` which is an immutable reference to a specific version of the build instructions that are responsible for signing.
```

Comment:
How should these attributes be supported by services like Buildkite which offer dynamic pipelines, i.e. build instructions are not immutable and change over the course of an execution? I suppose we could include a reference to the _initial_ build instructions?

Reply:
Ah, I see, the directory examples suggest this could be a link to the source repository at a specific commit sha. That doesn't represent the whole signing environment for us, the instructions also involve the running pipeline, but it might be close enough.

Reply:
Yes, so ideally this would be an immutable reference to the Buildkite pipeline steps at the time the build ran.

How are Buildkite pipeline steps stored / versioned? If I update the pipeline steps **after** a build has run, is there a URL I can go to that references the previous version of the pipeline steps (e.g. the pipeline steps when the build ran)?